### PR TITLE
adds grade distribution and styles graphs

### PR DESCRIPTION
**Pull request by hailed748**
- Adds grade distribution chart using the jStats library to compute distribution math
- Styles other graphs to have a nice color pallet
- Adds grade portion to data request

Note that modifications needs to be made on the form eventually to take in grade data.

image (5)

### CODE REVIEW

File: src/test/java/com/google/sps/servlets/DataServletTest.java [hunk 1]
```diff
@@ -1,152 +1,154 @@
-package com.google.sps.servlets;
```
Comment (ninaprabhu) [inline src/test/java/com/google/sps/servlets/DataServletTest.java:1]: if you're just commenting this out, maybe don't commit these changes

Comment (hailed748) [reply src/test/java/com/google/sps/servlets/DataServletTest.java]: Removed it!

File: src/main/webapp/course-style.css [hunk 2]
```diff
@@ -64,3 +64,22 @@
   font-family: monospace;
   font-size: 16px;
 }
+
+.container-chart {
+  width: 85%;
```
Comment (LukeGMTravis) [inline src/main/webapp/course-style.css:69]: Nit: Conventionally these are alphabetized

Comment (LukeGMTravis) [reply src/main/webapp/course-style.css:69]: Same elsewhere

File: src/main/webapp/courseLoader.js [hunk 3]
```diff
@@ -187,6 +242,98 @@
...
+    jData = jStat(data);
+    mean = jData.mean();
+    stndDev = jData.stdev();
+    xMin = jData.min();
+    xMax = jData.max();
+
+    let chartData = new Array([]);
+    let index = 0;
+    for (var i = xMin; i <= xMax; i += 0.01) {
+      chartData[index] = new Array(4);
```
Comment (LukeGMTravis) [inline src/main/webapp/courseLoader.js:329]: Why 4?

Comment (hailed748) [reply src/main/webapp/courseLoader.js:329]: There are four columns.

**Pull request merged**